### DISCUSSION TITLE
When will a new official relese be published?

### DISCUSSION THREAD

**Discussion by edu1010** [Facepunch/Facepunch.Steamworks#782] [Q&A]
The current relese is quite outdated with respect to the project itself, if there are no plans to publish more releses, I think that a small guide for the compilation of the library should be published, it would help a lot to keep the project in use by small developers.

**Comment by xzbobzx** [comment]
omg

https://github.com/Facepunch/Facepunch.Steamworks/releases/tag/2.4.1
